configure.ac

```diff
@@ -153,6 +153,13 @@ AS_IF([test "x$enable_tar_selinux" = "xyes"],
 	    [TARSELINUX=no]
 )
 
+AC_ARG_ENABLE(
+        [stateless],
+        AS_HELP_STRING([--disable-stateless],[OS is not stateless, do not ignore configuration files (stateless by default)]),
+        AC_DEFINE(OS_IS_STATELESS,0,[OS is not stateless]),
+        AC_DEFINE(OS_IS_STATELESS,1,[OS is stateless])
+)
```

Comment:
@phmccarty you are right, the enable_val needs to be checked in the first case. I'll fix that.